core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -150,7 +166,7 @@ class ConsoleConsumerTest extends JUnitSuite {
     val config = new ConsoleConsumer.ConsumerConfig(args)
 
     //Then
-    assertTrue(config.useNewConsumer)
+    assertFalse(config.useOldConsumer)
```

Comment:
The `useNewConsumer` boolean flag is removed.  `useOldConsumer` flag is used instead so the assertion is just inverted.